packages/webdoc-default-template/tmpl/pages/api-index.tmpl

```diff
@@ -1,29 +1,18 @@
 <?js
-    /* Renders a table of all the classes in the API surface */
-
-    /*::
-     * import type {ClassDoc} from "@webdoc/types";
-     * import type {Index} from "@webdoc/default-template/helper/renderer-plugins/index-sorter";
-     */
-    const { title, documentList } /*: { ClassDoc[] } */ = obj;
-
-    const classIndex /*: Index */ = this.plugins.generateIndex(documentList);
-  ?>
-    <h1 class="document-title"><?js= title ?></h1>
-    <hr />
-  <?js
-    classIndex.forEach(({ prefix, list}) => {
-  ?>
-    <h2 class="section-title"><?js= prefix ?></h2>
-    <hr />
-    <table>
-        <tbody>
-            <?js list.forEach((doc) => { ?>
-                <tr>
-                    <td><?js= this.linkTo(doc) ?></td>
-                    <td><?js= doc.brief || "" ?></td>
-                </tr>
-            <?js }); ?>
-        </tbody>
-    </table>
-<?js }); ?>
+const { title, documentList } = obj;
+const classIndex = this.plugins.generateIndex(documentList);
+?>
+<div class="document">
+  <h1 class="document__title"><?js= title ?></h1>
+  <div class="api-index">
+  <?js classIndex.forEach(({ prefix, list}) => { ?><h2 class="api-index__title"><?js= prefix ?></h2>
+  <table class="api-index-list">
```

Comment:
We should have indents inside the `<?js { ?> <?js } ?>`, and `<h2></h2>` should not be on the same line.

Reply:
I made this to produce more flat and readable HTML output.

Reply:
Also looking on html docs, tags with text (`p`, `h*`, `span`, `li`, etc.) are better being trimmed of whitespaces, but I'll fix this to meet your criteria.

Reply:
Fixed in 303e70a6ed60c787ce0ca7cf78706533b7d1e483.